src/sage/interfaces/axiom.py

```diff
@@ -458,14 +457,14 @@ def _eval_line(self, line, reformat=True, allow_use_file=False,
             return out
         # out = out.lstrip()
         i = out.find('\n')
-        out = out[i+1:]
+        out = out[i + 1:]
```

Comment:
I think this is incorrect per PEP 8 - https://peps.python.org/pep-0008/#pet-peeves
"in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority)."

Reply:
There is equal amounts of space on either side of `:`, namely, zero. I see no problem here.

Reply:
Ah, you meant
```
# Correct:
ham[lower+offset : upper+offset]
# Wrong:
ham[lower + offset:upper + offset]
```
You are right. 

But personally I cannot agree with the recommendation.